test/run_pass/issue/issue-1101.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var UART = require('uart');
+var uart = new UART();
+var res = uart.open({
+    device: '/dev/ttyS1',
```

Comment:
It would be better to use the same 2 spaces indentation rule in the tests too.

Reply:
I fixed it.